.github/config.yml

```
@@ -1,2 +1,2 @@
 newIssueWelcomeComment: >
-  Thank you for creating the issue! One of our team members will get back to you shortly with additional information.
+  Thank you for creating the issue! One of our team members will get back to you shortly with additional information. If this is a product issue, please close this and open and issue on https://developercommunity.visualstudio.com for the particular package.
```

```suggestion
  Thank you for creating the issue! One of our team members will get back to you shortly with additional information. If this is a product issue, please close this and contact the particular product's support instead (see https://support.microsoft.com/allproducts for the list of support websites).
```
Maybe this could work?

---

Thanks!